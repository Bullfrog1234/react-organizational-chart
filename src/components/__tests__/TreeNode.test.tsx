import React from "react";
import TreeNode from "../TreeNode";
import renderer from "react-test-renderer";

describe("TreeNode", () => {
  it("is exported", () => {
    expect(TreeNode).toBeTruthy();
  });

  it("Renders", () => {
    const component = renderer.create(<TreeNode label={"test"} />);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
