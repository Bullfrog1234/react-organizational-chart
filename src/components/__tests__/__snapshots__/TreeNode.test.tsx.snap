// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TreeNode Renders 1`] = `
<li
  className="css-1bmag6y"
>
  test
</li>
`;
